src/app/modules/OpenshiftStreams/dialogs/CreateInstance/api.ts

```diff
@@ -204,6 +204,10 @@ export const useCreateInstance = (): OnCreateKafka => {
           case ErrorCodes.INSUFFICIENT_QUOTA:
             onError('over-quota');
             break;
+          case ErrorCodes.REACHED_MAX_LIMIT_ALLOWED_KAFKA:
+          case ErrorCodes.INSTANCE_TYPE_NOT_SUPPORTED:
+            onError('trial-unavailable');
```

Comment:
[optional] While this is only key name it might be good to change it
```suggestion
            onError('instance-temporary-unavailable');
```

Reply:
Those strings are defined by the UI here https://github.com/redhat-developer/app-services-ui-components/blob/2cbea7d6821445c41dd461453f10397c80b2c29c/src/Kafka/CreateKafkaInstance/machines/types.ts#L24 and are mapped to specific messages shown to the user. I agree on the change tho, unless we need to give a different message to paying users than what we give to trial users. Cc @jgiardino

Reply:
This is probably one case where the image we created for trial users would also work for paying users. If I'm following this case correctly, this is the message we currently display for trial users:
> Kafka instances are unavailable for creation
> Kafka instances are no longer available. Try again in a few hours.

Reply:
Agree. Let's merge this and revisit the cases on the ui-components repo.

Reply:
> This is probably one case where the message we created for trial users would also work for paying users. If I'm following this case correctly, this is the message we currently display for trial users:
> 
> > Kafka instances are unavailable for creation
> > Kafka instances are no longer available. Try again in a few hours.

Yes, we are already handling message for standard/paying users.
>>Kafka instances are unavailable for creation
>>Your organization reached the limit of Kakfa instances available for creation.